queue_services/entity-filer/tests/unit/test_worker/test_correction_special_resolution.py

```diff
@@ -117,6 +118,12 @@ def __init__(self, file_content):
         assert party.first_name == 'JOEY', 'First name should be corrected'
         assert party.last_name == 'DOE', 'Last name should be corrected'
 
+        # Check outcome
+        final_filing = Filing.find_by_id(correction_filing_id)
+        alteration = final_filing.meta_data.get('correction', {})
+        assert business.association_type == coop_associate_type
+        assert alteration.get('toCooperativeAssociationType') == coop_associate_type
+
```

Comment:
can we confirm `fromCooperativeAssociationType` is the expected value too?

Reply:
yes, I just added